src/legacy/core_plugins/data/public/filter/filter_manager/filter_manager.ts

```diff
@@ -0,0 +1,245 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Filter, isFilterPinned, FilterStateStore } from '@kbn/es-query';
+
+import _ from 'lodash';
+import { Subject, Subscription } from 'rxjs';
+
+import { npSetup } from 'ui/new_platform';
+
+// @ts-ignore
+import { compareFilters } from './lib/compare_filters';
+// @ts-ignore
+import { mapAndFlattenFilters } from './lib/map_and_flatten_filters';
+// @ts-ignore
+import { uniqFilters } from './lib/uniq_filters';
+// @ts-ignore
+import { extractTimeFilter } from './lib/extract_time_filter';
+// @ts-ignore
+import { changeTimeFilter } from './lib/change_time_filter';
+
+import { PartitionedFilters } from './partitioned_filters';
+import { FilterStateManager } from './filter_state_manager';
+
+// const COMPARATOR_OPTIONS = { negate: true, disabled: true };
+
+export class FilterManager {
+  filterState: FilterStateManager;
+  indexPatterns: any;
+  filters: Filter[] = [];
```

Comment:
should be private

Reply:
Do you have any examples of other places in the code that do this?
I read up a little and saw this about private class fields.
https://github.com/tc39/proposal-private-fields/blob/master/FAQ.md

Reply:
we don't want this to be part of public interface of filterManager
just search code for `private ` there is quite a lot of it around, for example in embeddables api